superset-frontend/src/explore/controlUtils.js

```diff
@@ -193,20 +193,25 @@ const getMemoizedSectionsToRender = memoizeOne(
       }
     });
 
-    const { datasourceAndVizType, sqlaTimeSeries, druidTimeSeries } = sections;
-    const timeSection =
-      datasourceType === 'table' ? sqlaTimeSeries : druidTimeSeries;
+    const { datasourceAndVizType } = sections;
+    // list of datasource-specific controls that should be removed
+    const invalidControls =
```

Comment:
I saw in attached PR #869 also `time_range`, shouldn't be it here?

Reply:
`time_range` is the exception to the rule, as it should always be available (this will be noved to the forthcoming Filters section).